test/gtest/uct/cuda/test_cuda_ipc_md.cc

```diff
@@ -67,6 +122,34 @@ UCS_MT_TEST_P(test_cuda_ipc_md, multiple_mds, 8)
         rkey = unpack(md, i + 1);
         EXPECT_EQ(i, rkey.dev_num);
     }
+
+#if HAVE_CUDA_FABRIC
```

Comment:
maybe create a separate test function for mempool allocations?

Reply:
done